### ISSUE TITLE
Top "KeyRunner" menu > "Check for Updates" functionality is broken

### ISSUE THREAD

**Issue by frank-duq** [launchiamenterprise/keyrunner#64]
**Describe the bug**
Running on KeyRunner `1.0.65`, when trying the "Check for Updates" functionality, it appears to be broken.

**To Reproduce**
1. Go to "KeyRunner" top menu
2. Select "Check for Updates"
3. See dialog error ("Error during the update"   -->    "Error:  options.name must be either a string or a Buffer")

**Expected behavior**
"Check for Updates" feature is not throwing an error, but rather, either telling no available updates (or "You are currently running the latest version of KeyRunner"), or providing me with an update if there is one.

**Screenshots**
![Screenshot 2024-10-09 at 4 31 44 PM](https://github.com/user-attachments/assets/7604b20f-c5be-48d0-953a-3f9b0a059bbf)

![Screenshot 2024-10-09 at 4 31 56 PM](https://github.com/user-attachments/assets/2e986427-ca56-42f9-9548-d9054b60b2df)

**Desktop (please complete the following information):**
 - OS: [macos]
 - Version [sonoma]

**Comment by launchiamenterprise** [comment]
@frank-duq Could you please try the latest version and let us know if the issue persists?

**Comment by frank-duq** [comment]
@launchiamenterprise yup, error still persists with latest `1.0.69`, could be due me using macos (silicon)...

Let me know if there's any internal app logs in some file somewhere on filesystem's internal keyrunner storage that could be of any help for you..

**Comment by launchiamenterprise** [comment]
@frank-duq we were able to reproduce this issue in one of our machines (Latest version of silicon) and it seems to occur only on that machine. We are still analyzing this issue.

**Comment by launchiamenterprise** [comment]
@frank-duq this has been fixed with V#1.0.76 . Please update to latest
